src/asthaSrivastava/BankOperations.java

```diff
@@ -0,0 +1,85 @@
+package asthaSrivastava;
+
+public class BankOperations {
+	int balance;
+	static int debitCount;
+	static int creditCount;
+	static int currentBalance;
+	int iDebitCount;
+	int iCreditCount;
+	int iCurrentBalance;
+	
+	void setUserDetails(String name, int iBal) {
+		balance= iBal;
+		System.out.println("Username is "+name );
+		System.out.println("Inital Balance is :"+balance );		
+	}
+	
+	void debitOp(int num) {
+		balance = balance - num;
```

Comment:
please add if condition like

if(balance > num){
    //code
}

Reply:
Hi Ma'am,

Do I need to update this file in local and do again git add, commit and push?
As I can see this has been merged into master.

Please confirm

Thanks,
Astha